src/Features/CSharp/Portable/CodeRefactorings/InvertIf/InvertIfCodeRefactoringProvider.cs

```diff
@@ -63,32 +64,125 @@ internal partial class InvertIfCodeRefactoringProvider : CodeRefactoringProvider
                 return;
             }
 
+            var invertedIfBody = SyntaxKind.None;
+            if (ifStatement.Else == null)
+            {
+                var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+                if (!CanInvertElselessIf())
+                {
+                    return;
+                }
+
+                bool CanInvertElselessIf()
+                {
+                    var controlFlow = semanticModel.AnalyzeControlFlow(ifStatement.Statement);
+                    return !controlFlow.EndPointIsReachable || (invertedIfBody = GetInvertedIfGeneretedBody()) != default;
```

Comment:
So, i find with control flow, it's helpful to comment to explain why this is necessary.  i.e. why you can only convert if the endpotin was not reachable.  why it woudlnt' be convertible if it was.  etc. etc.